packages/node-modules-inspector/src/app/components/panel/Filters.vue

```diff
@@ -66,6 +70,62 @@ function resetExcludes() {
 const moduleTypes = Object.fromEntries(
   MODULE_TYPES_FULL_SELECT.map(x => [x, createModuleTypeRef(x)] as const),
 ) as Record<PackageModuleType, WritableComputedRef<boolean>>
+
+watch(searchText, (v) => {
+  if (filters.mode === 'text') {
+    const authorMatches = [...v.matchAll(FILTER_AUTHOR_REGEX)].map(m => m[1])
```

Comment:
I think we should extra this into a pure utility and do some unit tests about the parsing.